src/components/OcButton.spec.js

```diff
@@ -0,0 +1,181 @@
+import OcButton from './OcButton.vue';
+import {createLocalVue, shallowMount} from '@vue/test-utils';
+const localVue = createLocalVue()
+
+// this call will be set before every test
```

Comment:
I think it's clear what `beforeEach` does, a better comment would be to explain what you are trying to achieve here